buildSrc/build.gradle

```diff
@@ -17,39 +17,39 @@
  */
 
 plugins {
-  id 'java'
+    id 'java'
```

Comment:
Looks like only indentation changed here. I guess `spotless` can't run on the `buildSrc` dir. Can you manually fix it?